ui/src/app/shared/services/group-route.service.ts

```diff
@@ -13,10 +15,7 @@ export class GroupRouteService {
   }
 
   create(args): string {
-    const key = `group-${'xxxxx-xxxxx-xxxxx-xxxxx'.replace(/[xy]/g, function (c) {
-      const r = Math.random() * 16 | 0, v = c === 'x' ? r : (r & 0x3 | 0x8);
-      return v.toString(16);
-    })}`;
+    const key = uuidv4();
```

Comment:
The UUID should respect the following format: `xxxxx-xxxxx-xxxxx-xxxxx`
The `isSimilar` method is used to define if the GET params is a stream definition or a group (UUID).